testsuite/MDAnalysisTests/analysis/test_wbridge.py

```diff
@@ -17,6 +17,26 @@ def test_import_from_hbonds():
                              "importing WaterBridgeAnalysis from "
                              "MDAnalysis.analysis.hbonds failed.'")
 
+
+def test_import_warning():
+    with pytest.warns(DeprecationWarning) as record:
```

Comment:
Given my other comment, rather than explicitly testing the record, it's probably easier/cleaner to just check using the `match` keyword here:
```suggestion
    wmsg = 'Please use MDAnalysis.analysis.hydrogenbonds.wbridge_analysis'

    with pytest.warns(DeprecationWarning, match=wmsg):
```